build.gradle

```diff
@@ -57,18 +84,19 @@ embulkPlugin {
     type = "jsonpath"
 }
 
-publishing {
-    publications {
-        embulkPluginMaven(MavenPublication) {  // Publish it with "publishEmbulkPluginMavenPublicationToMavenRepository".
-            from components.java  // Must be "components.java". The dependency modification works only for it.
-        }
-    }
-    repositories {
-        maven {
-            url = "${project.buildDir}/mavenPublishLocal"
-        }
-    }
-}
+// TODO: enable when the pure-java plugin publish.
+//publishing {
+//    publications {
+//        embulkPluginMaven(MavenPublication) {  // Publish it with "publishEmbulkPluginMavenPublicationToMavenRepository".
+//            from components.java  // Must be "components.java". The dependency modification works only for it.
+//        }
+//    }
+//    repositories {
+//        maven {
+//            url = "${project.buildDir}/mavenPublishLocal"
+//        }
+//    }
+//}
```

Comment:
I don't think you need to comment it out. You can just keep this section, and you can check it for example by `./gradlew -Dmaven.repo.local=build/mavenLocal publishToMavenLocal`

(It generates Maven files under `embulk-parser-jsonpath/build/mavenLocal/`.)